app/styles/_global.scss

```diff
@@ -206,3 +254,159 @@ fieldset[disabled] .form-control {
         width: 1170px;
     }
 }
+
+// Bootstrap Navbar
+
+.navbar-toggle {
+    position: relative;
+    float: right;
+    padding: 9px 10px;
+    margin-right: 15px;
+    margin-top: 8px;
+    margin-bottom: 8px;
+    background-color: transparent;
+    background-image: none;
+    border: 1px solid transparent;
+    border-radius: 4px;
+}
+
+.navbar-toggle:focus {
+    outline: 0;
+}
+
+.navbar-toggle .icon-bar {
+    display: block;
+    width: 22px;
+    height: 2px;
+    border-radius: 1px;
+}
+
+.navbar-toggle .icon-bar+.icon-bar {
+    margin-top: 4px;
+}
+
+.nav {
+    padding-left: 0;
+    margin-bottom: 0;
+    margin-top: 0;
+    list-style: none;
+}
+
+.nav::after,
+.nav::before {
+    display: table;
+    content: ' ';
+    clear: left;
```

Comment:
ditto with this `clear: left`, I don't see it in the bundled css file, but doesn't seem to change any layout. Although there is a `.nav::after { clear: both; }` that isn't present in this PR?

Reply:
You would see this effect by going to the dashboard page, making it mobile sized, then pulling down the secondary nav from the navbar. The "My projects" item would be indented without this. I didn't need to clear both because only the left side seemed affected.

Reply:
Although it may also have been fixed by all the global element css I added in later.